src/Chapter1.hs

```diff
@@ -652,8 +667,13 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit = (positiveFirstDigit . abs)
```

Comment:
```suggestion
firstDigit = positiveFirstDigit . abs
```

Reply:
Ack, I also shadowed the `elem` fn here it seems.